pyproject.toml

```diff
@@ -34,3 +34,7 @@ include = ["zinolib"]
 
 [tool.setuptools_scm]
 write_to = "src/zinolib/version.py"
+
+[tool.isort]
+profile = "black"
+src_paths = ["src", "tests"]
```

Comment:
Why "src_paths"? It works fine without it.

This setup alters just about every single file on the first run. Therefore, the commit hash of a first global run needs to be added to `.git-blame-ignore-revs`.